### ISSUE TITLE
duplicate property name 'title' with other package?

### ISSUE THREAD

**Issue by thearabbit** [monbro/meteor-breadcrumb-plugin#5]
I used other package that use `title` property, so it duplication with this.
how to solve.

**Comment by monbro** [comment]
You are right, it should be namespaced. Thank you for reporting, I will adjust this in the next release.

**Comment by thearabbit** [comment]
can't wait

**Comment by stephane-r** [comment]
Hi,

I've add pull request for change 'title' and add microdata : https://github.com/monbro/meteor-breadcrumb-plugin/pull/17

On your route, replace 'title' by 'itemName'.

I was conflict with ms-seo (used iron:router and 'title'). Worked for me.
